grpccache.go

```diff
@@ -164,6 +169,31 @@ func (c *Cache) Store(ctx context.Context, method string, arg proto.Message, res
 	return nil
 }
 
+// cleanOldEntries removes expired entries from the cache. It does so in a
+// separate goroutine and only once every 10 seconds. The method assumes that
+// the cache mutex is locked.
+func (c *Cache) cleanOldEntries() {
+	now := time.Now()
+	if now.After(c.lastClean.Add(10 * time.Second)) {
+		c.lastClean = now
```

Comment:
I missed that. It looks good.

> The method assumes that the cache mutex is locked.

I've seen this better phrased as "The caller must be holding the Cache.mu lock."